2-projects/1-todo-list/script-NonFontAwesome.js

```diff
@@ -0,0 +1,133 @@
+function addDateElement() {
+  const rightFormEl = document.querySelector(".col-auto");
+  const createLabel = document.createElement("label");
+  createLabel.htmlFor = "endDate";
+  createLabel.textContent = "End Date:";
+  const createInput = document.createElement("input");
+  createInput.type = "date";
+  createInput.id = "endDate";
+  createInput.name = "todo-end";
+  createInput.min = new Date().toISOString().slice(0, 10);
+  const createBtn = document.createElement("button");
+  createBtn.type = "button";
+  createBtn.classList = "btn-close";
+  createBtn.id = "endDateCloseBtn";
+  rightFormEl.append(createLabel, createInput, createBtn);
+}
+
+addDateElement();
+
+const list = document.getElementById("todo-list");
+const endDateEl = document.querySelector("#endDate");
+const todoInputEl = document.querySelector("#todoInput");
```

Comment:
Please group the code outside the functions together, either put them at the top (while functions at the bottom) or at the bottom (functions at the top). This improves readability.